<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefNetcastDlg</class>
 <widget class="QWidget" name="DlgPrefNetcastDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form3</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="enableHeadphoneBroadcast">
     <property name="text">
      <string>Enable LAN headphone broadcast</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Audio Out Connection</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxHost">
        <item>
         <property name="text">
          <string>Broadcast</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="port">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>4596</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProtocolType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>UDP/IP</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>483</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_16">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Please note that for technical reasons any change in the settings may cause a short stall in the sound.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Encoding</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Bitrate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEncodingBitrate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>320 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>256 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>224 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>192 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>160 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>128 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>112 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>96 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>80 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>64 kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>48 kbps</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEncodingFormat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>MP3</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Channels</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEncodingChannels">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Stereo</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Stream Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Auto flush</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxAutoFlush">
        <item>
         <property name="text">
          <string>Silence detected</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8 seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 second</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Half second</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every frame</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>enableHeadphoneBroadcast</tabstop>
  <tabstop>comboBoxProtocolType</tabstop>
  <tabstop>host</tabstop>
  <tabstop>port</tabstop>
  <tabstop>comboBoxEncodingBitrate</tabstop>
  <tabstop>comboBoxEncodingFormat</tabstop>
  <tabstop>comboBoxEncodingChannels</tabstop>
  <tabstop>comboBoxAutoFlush</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>enableHeadphoneBroadcast</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DlgPrefNetcastDlg</receiver>
   <slot>enableGroupBoxes(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableGroupBoxes(int)</slot>
 </slots>
</ui>
